frontend/src/components/LeaderBoard.tsx

```
@@ -0,0 +1,248 @@
+                              <div className="flex items-center gap-1 border border-neutral-400 px-2 py-1 rounded-md">
+                                <span className="text-sm">{feed.feedId}</span>
+                                <span className="text-sm">{feed.count}</span>
+                              </div>
+                            </div>
+                          ))}
+                        </div>
+                      )}
+                    </div>
+                  </td>
+                </tr>
+              ))}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)